benchmark_test.go

```diff
@@ -1,105 +0,0 @@
-package containerd
```

Comment:
We were not running the benchmarks in CI and we later added the `containerd-stress` tool that is doing this on an instance.